x-pack/test/functional/apps/upgrade_assistant/es_deprecation_logs_page.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function upgradeAssistantESDeprecationLogsPageFunctionalTests({
+  getService,
+  getPageObjects,
+}: FtrProviderContext) {
+  const PageObjects = getPageObjects(['upgradeAssistant', 'common']);
+  const retry = getService('retry');
+  const security = getService('security');
+  const testSubjects = getService('testSubjects');
+  const es = getService('es');
+
+  describe('ES deprecation logs page', function () {
+    this.tags('skipFirefox');
+
+    before(async () => {
+      await security.testUser.setRoles(['superuser']);
+      // Access to system indices will be deprecated and should generate a deprecation log
+      await es.indices.get({ index: '.kibana' });
+    });
+
+    after(async () => {
+      await security.testUser.restoreDefaults();
+    });
+
+    beforeEach(async () => {
+      await PageObjects.upgradeAssistant.navigateToFixDeprecationLogs();
+
+      // Only click deprecation logging toggle if its not already enabled
+      if (!(await testSubjects.isDisplayed('externalLinksTitle'))) {
```

Comment:
`externalLinksTitle` does not seem to reference the actual DOM node. It seems to be the title of the page "Analyze deprecation logs". Can we update it?

Reply:
Ahhh re-reading this `if` block, i think it can be safely deleted since deprecation toggle is enabled by default now. This was used before when we had to keep it enabled for all tests.